finance/server/src/main/java/com/google/finapp/schema_pg.sdl

```diff
@@ -48,4 +48,8 @@ CREATE TABLE CustomerRole (
     REFERENCES Account(AccountId)
 ) INTERLEAVE IN PARENT Customer ON DELETE CASCADE;
 
+CREATE TABLE CloudSpannerSampleApp (
+  id INT64,
+) PRIMARY KEY (id);
+
```

Comment:
This should be postgres DDL, see above:
- INT64 -> BIGINT
- PRIMARY KEY definition should be inside the '(' ')'